### DISCUSSION TITLE
Where are default WPF controls templates?

### DISCUSSION THREAD

**Discussion by maxima120** [dotnet/wpf#7674] [Q&A] [answered]
In older version I would use Blend and extract a copy of a default template, edit it and use. Now "Create a Copy" is disabled (tried Blend and VS2022).

I found this page: https://learn.microsoft.com/en-us/dotnet/desktop/wpf/controls/control-styles-and-templates?view=netframeworkdesktop-4.8&viewFallbackFrom=netdesktop-7.0

But it is for .Net 4x and I dont really trust it would be 100% accurate - I need a definitive solution. Any advice?

**Comment by miloush** [comment, marked as answer]
Are you looking for [src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.AeroLite/Themes/AeroLite.NormalColor.xaml](https://github.com/dotnet/wpf/blob/cda43cae95e6110042bb69b7871a1e0ac927aa69/src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.AeroLite/Themes/AeroLite.NormalColor.xaml)?

All the themes are there.
